docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+
+|Name| Description  | Color | Example
+|---|---|---|------|
+|Draft| The issue definition is still being worked on and it is not ready to start development. Once the issue is ready the status should be changed to `Approved` or `Backlog`.| ![#ededed](https://via.placeholder.com/15/ededed/000000?text=+) `#ededed` | `Status:Draft` |
+|New| The issue has just been created and it has not been reviewed by the team. Once the issue is reviewed the status be changed to `Approved`, `Backlog`, `Needs information` or just closed.| ![#bfd4f2](https://via.placeholder.com/15/bfd4f2/000000?text=+) `#bfd4f2` | `Status:New` |
+|Needs information| The issue as described is incomplete or not well understood. It is waiting for further information before it can continue.| ![#ff8c00](https://via.placeholder.com/15/ff8c00/000000?text=+) `#ff8c00` | `Status:Needs information` |
+|Backlog| The issue is out of scope for the current iteration but it will be evaluated in a future release. | ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Status:Backlog` |
+|Approved| The issue has been reviewed and is ready to start working on it, it will be added to the work queue in the current iteration. | ![#0e8a16](https://via.placeholder.com/15/0e8a16/000000?text=+) `#0e8a16` | `Status:Approved` |
+|Blocked| Current progress is blocked on something else. | ![#ff8c00](https://via.placeholder.com/15/FF8C00/000000?text=+) `#ff8c00` | `Status:Blocked` |
+|Stale| The issue hasn't been updated in a long time and will be automatically closed. | ![#ededed](https://via.placeholder.com/15/ededed/000000?text=+) `#ededed` | `Status:Stale` |
+
+### Community
+
+Use these labels to tag issues that involve the community.
+
+Color: All the labels in this category should use ![#874faf](https://via.placeholder.com/15/874faf/000000?text=+) `#874faf`.
+
+|Name| Description  | Example
+|---|---|------|
+|Help wanted| This is a good issue for a contributor to take on and submit a solution | `Community:Help wanted` |
+
+### Type
+
+Use these labels for providing additional information type of the issue.
+
+Color: This subcategory uses different colors for each label.
+
+|Name| Description  | Color | Example
+|---|---|---|------|
+|Bug| Your classic code defect.| ![#d73a4a](https://via.placeholder.com/15/d73a4a/000000?text=+) `#d73a4a` | `Type:Bug` |
+|Feature request| This is a new feature or an enhancement to an existing one.| ![#8f31ed](https://via.placeholder.com/15/8f31ed/000000?text=+) `#8f31ed` | `Type:Feature request` |
+|Customer ask| An enhancement or feature requested by a customer (use this label if the feature request was originated by a customer).| ![#8f31ed](https://via.placeholder.com/15/8f31ed/000000?text=+) `#8f31ed` | `Type:Customer ask` |
+|Parity| The issue describes a gap in parity between two or more platforms.| ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Type:Parity` |
```

Comment:
Since we have the "parity with XXX" labels below, how useful will this label be?

Reply:
"Automation: Parity with Xyz" is a tag meant to be applied to PRs to trigger some automation workflow (like creating child issues on a separate repo).

"Type:Parity" is meant to be applied to an issue that was found where something is in one platform and not in another.